elixir-ls-release/debugger.exs

```diff
@@ -0,0 +1,21 @@
+# Mix.install([
+#   {:elixir_ls, github: "elixir-lsp/elixir-ls", ref: "7f7ba8ac7ad518945f7622369d13c16f48568b30"}
+# ])
```

Comment:
remove?

Reply:
I will remove/clean this up. I need to find a way to easily switch between running local and published version